spring-cloud-gcp-docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -95,6 +97,13 @@ to be configured.
 
 `PubSubTemplate` uses a special subscriber generated by its `SubscriberFactory` to pull messages.
 
+If the message payload contains a serialized POJO, it can be retrieved as a `Class` compatible with that serialized payload:
```

Comment:
Maybe it's just me, but in the absence of additional context, the word "serialization" means the Java built-in serialization. So, to avoid confusion, I would instead use "JSON-encoded" or "JSON-serialized".

Reply:
I didn't think it was appropriate to reveal to the user specifically how we serialize their objects. Therefore I  didn't want to mention JSON/not-java/binary serialization specifically. I think "serialized" has just the connotation of sending an object over the network and not specifically Java Serializable serialization.

Reply:
It's important because the receiving end might not be the same application / code base. So, it will need to deal with deserialization.

Reply:
done.